webRequestTranslator/source/webRequestTranslator.cc

```diff
@@ -0,0 +1,189 @@
+#include "webRequestTranslator.h"
+
+namespace webRequestTranslator {
+    /**
+     * Converts the RDFS namenode create response into the appropriate webRDFS response.
+     */
+    std::string getNamenodeCreateResponse(hadoop::hdfs::DatanodeInfoProto &dataProto, std::string requestLink) {
+      std::string res = std::string("HTTP/1.1 307 TEMPORARY_REDIRECT\n");
+
+      std::string delimiter = "/webhfs/v1/";
+      std::string restOfRequest = requestLink.substr(requestLink.find(delimiter) + delimiter.length(), requestLink.length());
+
+      hadoop::hdfs::DatanodeIDProto id = dataProto.id();
+      res += "Location: http://";
```

Comment:
Do we have a solution for choosing between `http` and `https?` Maybe define it as an env var and conditionally choose one here based on that? cc @danielbalkum